docs/CHANGELOG.md

```diff
@@ -1,47 +1,79 @@
-
-Community:
-* Allow running tests on PRs from forks + label [#179](https://github.com/astronomer/astro-sdk/issues/179)
-
-Breaking Change:
-* The `aql.dataframe` argument `identifiers_as_lower` (which was `boolean`, with default set to `False`) was replaced by the argument `columns_names_capitalization` (`string` within possible values `["upper", "lower", "original"]`, default is `lower`).[#564](https://github.com/astronomer/astro-sdk/issues/564)
-* The `aql.load_file` before would change the capitalization of all column titles to be uppercase, by default, now it makes them lowercase, by default. The old behaviour can be achieved by using the argument `columns_names_capitalization="upper"`. [#564](https://github.com/astronomer/astro-sdk/issues/564)
-* `aql.load_file` attempts to load files to BigQuery and Snowflake by using native methods, which may have pre-requirements to work. To disable this mode, use the argument `use_native_support=False` in `aql.load_file`. [#557](https://github.com/astronomer/astro-sdk/issues/557), [#481](https://github.com/astronomer/astro-sdk/issues/481)
-* `aql.dataframe` will raise an exception if the default Airflow XCom backend is being used. To solve this, either use an [external XCom backend, such as S3 or GCS](https://www.astronomer.io/guides/custom-xcom-backends) or set the configuration `AIRFLOW__ASTRO_SDK__DATAFRAME_ALLOW_UNSAFE_STORAGE=True`. [#444](https://github.com/astronomer/astro-sdk/issues/444)
-* Change the declaration for the default Astro SDK temporary schema from using `AIRFLOW__ASTRO__SQL_SCHEMA` to `AIRFLOW__ASTRO_SDK__SQL_SCHEMA` [#503](https://github.com/astronomer/astro-sdk/issues/503)
+    - `Native`: Default, uses [Bigquery Load Job](https://cloud.google.com/bigquery/docs/loading-data) in the
+       case of BigQuery and Snowflake [COPY INTO](https://docs.snowflake.com/en/sql-reference/sql/copy-into-table.html)
+       using [external stage](https://docs.snowflake.com/en/sql-reference/sql/create-stage.html) in the case of Snowflake.
+    - `Pandas`: This is how datasets were previously loaded. To enable this mode, use the argument
+      `use_native_support=False` in `aql.load_file`.
+    - `Hybrid`: This attempts to use the native strategy to load a file to the database and if native strategy(i)
+       fails , fallback to Pandas (ii) with relevant log warnings.
+* Allow users to specify the table schema (column types) in which a file is being loaded by using `table.columns`.
+  If this table attribute is not set, the Astro SDK still tries to infer the schema by using Pandas
+  (which is previous behaviour).[#532](https://github.com/astronomer/astro-sdk/issues/532)
+* Implement fallback mechanism in case native support fails to default option with log warning for problem with native
```

Comment:
@kaxil we already talked about this in the `Hybrid` section, do we need to add this again?

Reply:
Good point, we should remove it